deploy.yaml

```
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: secrethub-mutating-webhook
+  labels:
+    app: secrethub-mutating-webhook
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: secrethub-mutating-webhook
+  template:
+    metadata:
+      labels:
+        app: secrethub-mutating-webhook
+    spec:
+      containers:
+      - name: secrethub-mutating-webhook
+        image: simonbarendse/kubernetes-mutating-webhook:latest
```

This is the image resulting from the `Dockerfile` in this repo. If we agree with the current state, let's push that to the `secrethub` org on DockerHub:

```suggestion
        image: secrethub/kubernetes-mutating-webhook:latest
```